src/loaders/core_text.rs

```diff
@@ -550,16 +550,21 @@ impl Font {
         let core_graphics_size = CGSize::new(canvas.size.x() as f64, canvas.size.y() as f64);
         core_graphics_context.fill_rect(CGRect::new(&CG_ZERO_POINT, &core_graphics_size));
 
-        match rasterization_options {
-            RasterizationOptions::Bilevel => {
+        match rasterization_options.antialiasing_strategy {
+            AntialiasingStrategy::Bilevel => {
                 core_graphics_context.set_allows_font_smoothing(false);
                 core_graphics_context.set_should_smooth_fonts(false);
                 core_graphics_context.set_should_antialias(false);
             }
-            RasterizationOptions::GrayscaleAa | RasterizationOptions::SubpixelAa => {
+            AntialiasingStrategy::GrayscaleAa | AntialiasingStrategy::SubpixelAa => {
                 // FIXME(pcwalton): These shouldn't be handled the same!
+                if rasterization_options.use_thin_strokes {
+                    core_graphics_context.set_should_smooth_fonts(false);
```

Comment:
could you add a comment explaining why we don't smooth fonts when thin strokes is enabled?